web/app.js

```diff
@@ -419,10 +419,14 @@ var PDFViewerApplication = {
     var support = !!(doc.requestFullscreen || doc.mozRequestFullScreen ||
                      doc.webkitRequestFullScreen || doc.msRequestFullscreen);
 
+//#if MOZCENTRAL
```

Comment:
I'm thinking we can include the check above it the `#if` too. Can you make MOZCENTRAL section as `var support = document.fullscreenEnabled === false;`.